toolbox/build/venv-requirements.txt

```diff
@@ -8,3 +8,6 @@ pytest-xdist>=1.30.0
 pycodestyle>=2.5.0
 voluptuous>=0.11.7
 yamllint>=1.20.0
+flake8==3.7.7
+ansible-lint==4.1.0
+bashate
```

Comment:
Nit: worth keeping alphabetic order here? Should we specify minimum version for bashate?

Reply:
yeahp, exporting the requirements usually adds the version and sorted alphabetically.